Source/OxyPlot/PlotModel/PlotModel.Rendering.cs

```diff
@@ -433,6 +433,13 @@ private void RenderSeries(IRenderContext rc)
         /// <param name="rc">The render context.</param>
         private void RenderTitle(IRenderContext rc)
         {
+            OxySize? maxSize = null;
+
+            if (ClipTitle)
+            {
+                maxSize = new OxySize((this.TitleArea.Left + this.TitleArea.Right) * TitleClippingLength, double.MaxValue);
```

Comment:
I think this should be

                maxSize = new OxySize(this.TitleArea.Width * TitleClippingLength, double.MaxValue);